modules/etcd-increase-db.adoc

```diff
@@ -0,0 +1,180 @@
+installer-9-ci-ln-qkgs94t-72292-9clnd-master-0           0/1     Completed           0          36s
+etcd-guard-ci-ln-qkgs94t-72292-9clnd-master-0            0/1     Running             0          26m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Terminating         0          11m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Terminating         0          11m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Pending             0          0s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Init:1/3            0          1s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     Init:2/3            0          2s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  0/4     PodInitializing     0          3s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  3/4     Running             0          4s
+etcd-guard-ci-ln-qkgs94t-72292-9clnd-master-0            1/1     Running             0          26m
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  3/4     Running             0          20s
+etcd-ci-ln-qkgs94t-72292-9clnd-master-0                  4/4     Running             0          20s
+...
+----
+
+. Review the values by entering the following command:
++
+[source,terminal]
+----
+oc describe -n openshift-etcd pod/<ETCD_PODNAME> | grep "ETCD_QUOTA_BACKEND_BYTES"
+----
++
+The values might not have changed from the default value of `8`.
++
+.Example output
+[source,terminal]
+----
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+ETCD_QUOTA_BACKEND_BYTES:                               8589934592
+----
+
+[id="etcd-change-db-size-troubleshooting_{context}"]
+== Troubleshooting
+
+If you encounter issues when you try to increase the database size for etcd, the following troubleshooting steps might help.
+
+[id="etcd-troubleshoot-small-value_{context}"]
+=== Value is too small
+
+If the value that you specify is less than 8, you will see an error message. In the following example, the value of 5 is specified:
+
+[source,terminal]
+----
+oc patch etcd/cluster --type=merge -p '{"spec": {"backendQuotaGiB": 5}}'
+----
+
+The output shows an error:
+
+.Example output
+[source,terminal]
+----
+The Etcd "cluster" is invalid: 
+* spec.backendQuotaGiB: Invalid value: 5: spec.backendQuotaGiB in body should be greater than or equal to 8
+* spec.backendQuotaGiB: Invalid value: "integer": etcd backendQuotaGiB may not be decreased
+----
+
+To resolve this issue, specify an integer between 8 and 32.
+
+[id="etcd-troubleshoot-large-value_{context}"]
+=== Value is too large
+
+If the value that you specify is greater than 32, you will see an error message. In the following example, the value of 64 is specified:
```

Comment:
```suggestion
If the value that you specify is greater than `32`, you see the following error message:
```


- Should 32 use back ticks because it is a valid value for `backendQuotaGiB`? Same question for all similar instances of values. 
- I think a present tense would suffice but up to you!
- "In the following example, the value of 64 is specified:" can be removed in favor of an example command